tests/scripts/code-style

```diff
@@ -48,8 +48,10 @@ shift
 
 # Verify C++ code styling in MFEM
 cd $mfem_dir
+
 make distclean
-make style
+make style ASTYLE_BIN=${ASTYLE:-astyle}
```

Comment:
Why is this necessary?

Reply:
When I was using astyle in Gitlab, I wanted to specify the full path to the desired version on astyle.
This is now useless and has been reverted.